docs/docs/reference/server/config.yml.md

```diff
@@ -179,6 +179,42 @@ There are two ways to configure AWS: using an access key or using the default cr
     Using private subnets assumes that both the `dstack` server and users can access the configured VPC's private subnets.
     Additionally, private subnets must have outbound internet connectivity provided by NAT Gateway, Transit Gateway, or other mechanism.
 
+??? info "OS images"
+    By default, `dstack` uses its own [AMI :material-arrow-top-right-thin:{ .external }](https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/AMIs.html)
+    optimized for `dstack`.
+    To use your own or other third-party images, set the `os_images` property:
+
+    ```yaml
+    projects:
+      - name: main
+        backends:
+          - type: aws
+            creds:
+              type: default
+
+            os_images:
+              cpu:
+                name: my-ami-for-cpu-instances
+                owner: self
+                user: dstack
+              nvidia:
+                name: 'Some ThirdParty CUDA image'
+                owner: 123456789012
+                user: ubuntu
+    ```
+
+    Here, both `cpu` and `nvidia` properties are optional, but if the property is not set, you won´t be able to use the corresponding instance types.
+
+    The `name` is an AMI name.
+    The `owner` is either an AWS account ID (a 12-digit number) or a special value `self` indicating the current account.
+    The `user` specifies an OS user for instance provisioning.
+
+    !!! info "Image requirements"
+        * SSH server listening on port 22
+        * `user` with passwordless sudo access
+        * Docker is installed
```

Comment:
And enabled (`systemctl enable docker`). I think it's not enabled by default after installation

Reply:
At least on Debian, Docker installed from `https://download.docker.com/linux/` is enabled by default, moreover, it also uses socket activation

```
debian@debian:~$ systemctl stop docker.service
Warning: Stopping docker.service, but it can still be activated by:
  docker.socket
debian@debian:~$ systemctl is-active docker.service
inactive
debian@debian:~$ docker ps
CONTAINER ID   IMAGE     COMMAND   CREATED   STATUS    PORTS     NAMES
debian@debian:~$ systemctl is-active docker.service
active
```

Anyway, I think it's too obvious to mention if someone is already into custom OS images, and I would also avoid any specific instructions on how to check it/enable, as this is init-specific (there are still distributions that use init systems other than systemd :) )